string.php

```diff
@@ -1066,3 +1066,16 @@ function zero_pad($num){
 	$num='0'.$num;
 	return substr($num,strlen($num)-2);
 }
+
+function startsWith($haystack, $needle){

```

Comment:
These should use _ for separation rather than camel case, as they're functions not methods.